src/contracts/misc/overview.md

```diff
@@ -0,0 +1,39 @@
+This document describes the roles and responsibilities of the Gho Stewards.
+
+## Gho Steward Contracts
+
+These contracts each control different parameters related to GHO, GSM, and CCIP. They allow the Aave DAO and an approved Risk Council to change these parameters, according to set rules and configurations.
```

Comment:
We can say "related to GHO and its facilitators"

Reply:
Done